main.nf

```diff
@@ -74,10 +39,28 @@ workflow {
         params.outdir,
     )
 
+    def tools = params.tools.tokenize(",")
+    if (tools.contains("all")) {
```

Comment:
Trimming was not done by default, it is not a bad idea to include it, but documentation needs updating

Reply:
Done! Thanks :)